plugin/trino-opa/pom.xml

```diff
@@ -132,6 +139,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jetbrains</groupId>
+            <artifactId>annotations</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
IntelliJ suggested I added this to some functions in the tests that take a SQL query as a string argument. I've seen other test code using this (e.g. the Cassandra connector or different classes in `trino-testing`).

I didn't add it to all functions though, and once I add it everywhere I'd need to propagate `@Language("SQL")` to any variable containing a query in the tests.
Happy for me to add this everywhere in the tests or do you want me to remove it?

Reply:
Since the `StandaloneQueryRunner` uses it:
https://github.com/trinodb/trino/blob/74e3ffa65e18d2db7938d5e17b6036b8dac42efc/core/trino-main/src/main/java/io/trino/testing/StandaloneQueryRunner.java#L106

I'm going to add it to all the testing code in a separate commit. Happy to remove if not needed